OpenRA.Mods.Common/Traits/Power/CanPowerDown.cs

```diff
@@ -19,6 +19,11 @@ public class CanPowerDownInfo : UpgradableTraitInfo, ITraitInfo, Requires<PowerI
 		[Desc("Restore power when this trait is disabled.")]
 		public readonly bool CancelWhenDisabled = false;
 
+		public readonly string PoweroffImage = "poweroff";
```

Comment:
You use a different naming style for each option. Please be consistent!  My preference is `IndicatorImage`, `IndicatorSequence`, and `IndicatorPalette`.